server/routes.js

```diff
@@ -548,3 +548,34 @@ exports.mountQuintypeAt = function(app, mountAt) {
     }
   });
 };
+
+/**
+ * ampRoutes uses quintype-amp library to handle amp pages
+ * currently, only story pages with default styling are supported
+ */
+exports.ampRoutes = function(app, { opts } = {}) {
+  const { Story } = require("./api-client");
+  const { ampifyStory } = require("@quintype/amp");
+  getWithConfig(
+    app,
+    "/amp/story/*",
```

Comment:
please match /amp/story/*/:storyId or something intuitive. There is no need for the string replace after this

Reply:
Okay